src/vivarium_gates_nutrition_optimization_child/constants/data_keys.py

```diff
@@ -41,7 +41,9 @@ class __DiarrhealDiseases(NamedTuple):
     DURATION: TargetString = TargetString("cause.diarrheal_diseases.duration")
     PREVALENCE: TargetString = TargetString("cause.diarrheal_diseases.prevalence")
     INCIDENCE_RATE: TargetString = TargetString("cause.diarrheal_diseases.incidence_rate")
-    REMISSION_RATE: TargetString = TargetString("cause.diarrheal_diseases.remission_rate")
+    REMISSION_RATE: TargetString = TargetString(
+        "cause.susceptible_to_diarrheal_diseases.remission_rate"
```

Comment:
What happened here?

Reply:
After updating VPH, it expects the remission rate key in the artifact to look this. Is this unexpected behavior?

Reply:
No, this is not expected. We should talk about this.